packages/volto/cypress/tests/core/basic/actions.js

```diff
@@ -44,7 +46,7 @@ describe('actions Tests', () => {
       cy.get('a[class="icon-align-name"]').should(
         'have.attr',
         'href',
-        '/my-page-1/contents',
+        `${prefixPath}/my-page-1/contents`,
```

Comment:
Can't we do it more elegant by invoking the `baseUrl` from Cypress config?

Reply:
@sneridagh I started looking into this but that won't be possible. `baseURL` contains the URL with the domain. For example:

`http://localhost/foo`

But links in HTML are relative. For example:

`/foo/my-document`